### ISSUE TITLE
Add a feed?

### ISSUE THREAD

**Issue by tfausak** [ProofOfKeags/proofofkeags.com#2]
Hi! I am the editor of Haskell Weekly. I would like to be notified when you publish a new post, but as far as I can tell your site does not have a feed. Would you mind adding an Atom or RSS feed? Hakyll supports this out of the box: https://jaspervdj.be/hakyll/tutorials/05-snapshots-feeds.html. Thanks!

**Comment by CaptJakk** [comment]
Absolutely. I have never used RSS feeds before and I'd like to figure out what they are and how they work. I'll look into doing this quite soon. Thanks for featuring my post in the latest issue!
